### ISSUE TITLE
Add support for Resharper 2017.2

### ISSUE THREAD

**Issue by pfeigl** [asizikov/AsyncSuffix#52]
Please add support for Resharper 2017.2 (wave >=9).

Thanks :)

**Comment by zvirja** [comment]
@asizikov It has been one week since the release 😉 Do you have any plans to add compatibility?

**Comment by asizikov** [comment]
Yes, I do. Once I'm back from vacation trip I'll address that. Unless
someone else wants to take care of that update 🙂

**Comment by zvirja** [comment]
Thanks for the reply!

Wish you a great vacation :) Do you have some approx. estimations in case nobody else picks that up (e.g. week, month, etc)? :)

**Comment by zvirja** [comment]
Hi Anton,

Thanks for the reply!
Wish you a great vacation :) Do you have some approx. estimations in case
nobody else picks that up (e.g. week, month, etc)? :)

Best regards,
Alex

**Comment by asizikov** [comment]
ok, that didn't go as easy as I expected. R# SDK has some breaking changes. I need to spend a bit more time on it.

**Comment by zvirja** [comment]
Hi. 

Do you know whether you have any time in the nearby future to get back to this one? We need this extension to guarantee the quality of our projects and it's a pity that it's missing..

**Comment by asizikov** [comment]
As I mentioned above, I need to figure out how to deal with the breaking changes in the new SDK. Can't give you any estimates, sorry.

**Comment by zvirja** [comment]
Could clarify the exact place to look? I also own a R# plugin, so I could potentially take a look..

**Comment by asizikov** [comment]
the Rename dialog has been rewritten, and the reflection hack which I used before is not working anymore. I didn't spend much time on it yet. Later today I'll publish the WIP branch, any help would be appreciated :)

**Comment by zvirja** [comment]
Have you published the PR? I cannot find it..

**Comment by asizikov** [comment]
@zvirja here you go: https://github.com/asizikov/AsyncSuffix/pull/55
